### PR TITLE
Scanbox refactoring and docstring

### PR DESCRIPTION
**Pull request by h-mayorquin**
This PR contains three small corrections for sbx:
1) Added a docstring.
2) Refactored assertion errors with file paths and sampling frequency.
3) Changed some links of the documentation that were death and added new ones.

Btw, I think this functions were copied as it is from the changed links. Maybe something to be aware off.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,9 @@ Developed by [CatalystNeuro](http://catalystneuro.com/).
 ## Usage:
 ### Supported file types:
 #### Imaging
-1. HDF5
-2. TIFF
-3. STK
-4. FLI
```
Comment (h-mayorquin) [inline README.md:23]: I am not sure which formats are FLI and STK. Maybe removing this is wrong but I could not find them in the library.

**Comment by CodyCBakerPhD** [comment]
> Btw, I think this functions were copied as it is from the changed links. Maybe something to be aware off.

I'm guessing that's because their code is not published on PyPI - if we wanted to import it, we would have to do a `git` install.

File: README.md (hunk 1 above)
Comment (CodyCBakerPhD) [reply README.md:23]: @h-mayorquin These can be found implemented here: https://github.com/catalystneuro/roiextractors-gpl/blob/master/roiextractors_gpl/extractors/bioformatsimagingextractors/bioformatsimagingextractors.py

You are free to move them over to this project, but the dependencies *must* be imported lazily due to the licensing difference (never put GPL license in our requirements/setup). It's a secondary software that the user must have installed separately.

We'll see if we can dig up the sample data they were based on but given it's been 2+ years that may be difficult.

**Comment by h-mayorquin** [comment]
> > Btw, I think this functions were copied as it is from the changed links. Maybe something to be aware off.
> 
> I'm guessing that's because their code is not published on PyPI - if we wanted to import it, we would have to do a `git` install.

Makes sense, let me know if there is anything missing for merging.

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**